### ISSUE TITLE
log file can be clearer

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/thirtybees#1616]
I got a "Duplicate entry '0' for key 'PRIMARY'" error. Obviously some missing auto-increase due to an incomplete import. The question is from which table.

I don't find it at all in the server error log. So it is an improvement that the /log directory mentions it. However, the improvement is small. It says:
`*ERROR* 	2023/01/25 - 12:43:37: ThirtyBeesDatabaseException: Duplicate entry '0' for key 'PRIMARY' at line 502 in file classes/db/Db.php`

So I still don't know the table. This kind of error message should be more complete. Maybe mention the query for all database errors?

**Comment by musicpanda** [comment]
For other errors the same applies. I get for example the error: Property OrderDetail->product_ean13 is not valid at line 1095 in file classes/ObjectModel.php

This tells me nothing about where to look for the error. It would be nice to have a backtrace added.

**Comment by getdatakick** [comment]
There is already `collectlogs` module that can collect all information for developers, including context.

**Comment by musicpanda** [comment]
I thought that collectlogs just collected the server logs. But those logs are just as summary as those in the "log" directory. An additional problem is that I often (always?) don't find the problems reported in the log directory in  my server log,

**Comment by getdatakick** [comment]
No, collectlogs has nothing to do with server logs. It collects php issues at runtime, on application level.

Depending on server logs is indeed very tricky, as it depends on configuration of individual servrer components (apache / nginx / phpfpm). A lot of hosting companies have error reporting disabled by default to save space.

**Comment by musicpanda** [comment]
Then why does it ask for a cron?

**Comment by getdatakick** [comment]
> Then why does it ask for a cron?

![image](https://user-images.githubusercontent.com/34651851/224726233-095ddede-b212-4880-b889-91e450684a57.png)

**Comment by musicpanda** [comment]
I find the collectlogs puzzling. It nowhere tells you where you can find the list of errors.

**Comment by getdatakick** [comment]
Fair enough, we should create shortcut from module config page to admin controller. 

Error logs can be found in "Advanced Parameters > Error Logs"
